### ISSUE TITLE
Solr indexing not completing

### ISSUE THREAD

**Issue by patrickzurek** [eXtensibleCatalog/Drupal-Toolkit#90]
JIRA issue created by: rcook
Originally opened: 2012-10-22 10:29 AM

Issue body:

________________________________________
From: Péter Király [mailto:kirunews@gmail.com]
Sent: Fri 10/19/2012 3:46 PM
To: Cook, Randall; Wesley, MT; Arbelo, Ralph; Lindahl, David
Subject: status report
Hi all,

I tried several times to index the full record set without success.
When it almost reached (or maybe it even reached) the end of process
Solr stopped to work. The Solr log only contains a cryptic "Killed"
line at the end without any description. I never meet with it before.
The index became corrupted, and it is not possible to commit/optimize
the records. I guess the cause might be something outside of Drupal
Toolkit - maybe a Solr 3.3 issue, maybe an OS issue (memory? disk
space?). I will try two things:

1. indexing with Solr 3.6.1
2. Sending commits after every Nth document (N might be 100K or 1M) --
it requires code change, so t will be the second attempt.

I hope that one of them will solve the issue.

Péter

**Comment by patrickzurek** [comment]
JIRA Comment by user: rcook
JIRA Timestamp: 2012-10-22 10:29 AM

Comment body:

2012/10/19 Arbelo, Ralph rarbelo@library.rochester.edu:

> Which server are you running this on?
> it is http://192.17.55.226/xc-full-test/.

Péter

Notify [~admin] [~mwesley]

**Comment by patrickzurek** [comment]
JIRA Comment by user: rcook
JIRA Timestamp: 2012-10-22 10:59 AM

Comment body:

Peter or [~mwesley]  if there is a chance this is server side, please involve [~bgant]

**Comment by patrickzurek** [comment]
JIRA Comment by user: pkiraly
JIRA Timestamp: 2012-10-24 03:10 AM

Comment body:

I created a multicore 3.6.1 Solr setup at /xc/apache-solr-3.6.1/example/multicore/ (the current (full) Drupal's core is xc-full-test, the index is available at /xc/apache-solr-3.6.1/example/multicore/xc-full-test/data/index/, it is 25GB). I updated the site, applied the autocomplete patch, started reindexing. It went fine without errors. The total time of indexing is 5:41 including optimization.
